cpp/include/cuml/decomposition/tsvd_mg.hpp

```diff
@@ -37,7 +37,7 @@ namespace opg {
  */
 void fit(raft::handle_t& handle,
          MLCommon::Matrix::RankSizePair** rank_sizes,
-         size_t n_parts,
+         std::size_t n_parts,
```

Comment:
Same here as well- this should never grow past 32-bits (if it does then we have bigger problems).